### ISSUE TITLE
Peer feedback

### ISSUE THREAD

**Issue by kristinawright** [STAT547-UBC-2019-20/group_12_qiyangqd_xiaoyuanf#38]
For the usage section, it would be much easier to use the scripts if the arguments that the user inputs were already filled out so that users do not have to find a file and file name to use, especially if one is to use this without knowing about the project structure. 
Ex.
`Rscript scripts/data_wrangle.R --raw_path=<raw_data_path> --clean_path=<clean_data_path>`
to 
`Rscript scripts/data_wrangle.R --raw_path=data/raw_data.csv --clean_path=data/cleaned_data.csv`

I like that the links to each of the r scripts are included for easy accessibility in the usage section. 

I think it would be a better fit to put the model.rds file in the data folder rather than the docs folder.

running: `Rscript scripts/knit.R --rmd_path=docs/finalreport.Rmd` throws me the error
`/Applications/RStudio.app/Contents/MacOS/pandoc/pandoc +RTS -K512m -RTS finalreport.utf8.md --to latex --from markdown+autolink_bare_uris+tex_math_single_backslash --output finalreport.tex --self-contained --highlight-style tango --pdf-engine xelatex --variable graphics --lua-filter`
`/Library/Frameworks/R.framework/Versions/3.6/Resources/library/rmarkdown/rmd/lua/pagebreak.lua --lua-filter` `/Library/Frameworks/R.framework/Versions/3.6/Resources/library/rmarkdown/rmd/lua/latex-div.lua --variable 'geometry:margin=1in'`
`! Package ifvtex Error: Name clash, \ifvtex is already defined.`

`Error: LaTeX failed to compile finalreport.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See finalreport.log for moreinfo.`
`Execution halted`
It did not produce an output pdf file.

Looking at the pdf file already created, Figure 1 has the figure title cut off and Figure 3 plus Figure 5 have tiny text that is hard to read.  Other than that it looks clean and understandable for the code and written report!  Your analysis was interesting in that you found PM2.5 concentration in Beijing changed based on wind direction!

**Comment by qiyangqd** [comment]
Hi @kristinawright ! Thanks a lot for your suggestions! We'll make the project more accessible to users. About the LaTeX compiling error, could you let me know if you were able to compile other .pdf before? Or could you kindly send me the `finalreport.log` so I can try to find a solution? I searched online for the error `! Package ifluatex Error: Name clash, \ifluatex is already defined`, and most of the cases were about local LaTeX versions. But there might also be something wrong with the codes, so I would like to check it. Thanks and take care!

**Comment by qiyangqd** [comment]
Hi @kristinawright ! Thanks again for your suggestions! We have addressed most of them specified as below:
1. Specified `Usage` in README.
2. Removed `.rds` to the `data` folder.
3. Fixed the problem that the title of the correllogram is cut off.
4. Increased the size of texts in all figures that had tiny texts (thanks for the reminder!).
5. About failing to knit a `.pdf` of the final report: we are sorry you wasn't able to generate a `.pdf` as we succeeded in doing so on our computers. Thus, we are speculating this might have something to do with local packages or settings. [Here](https://tex.stackexchange.com/questions/515598/package-ifluatex-error-name-clash-ifluatex-is-already-defined) is a link I found that might be useful: the person who posted the issue solved it by "updated the "MikTex Console" in adm and user mode". We would also like to move this issue to __Future work__—let's do some trouble-shooting and see if we can fix it!
